infrastructure/modules/migration/envs/dev/configs/table_mappings.json

```diff
@@ -0,0 +1,27 @@
+{
+    "rules": [
+        {
+            "rule-id": 2122718802,
+            "rule-name": "mhaqg2qt-2b2v60",
+            "rule-type": "selection",
+            "rule-action": "include",
+            "object-locator": {
+                "schema-name": "npd",
+                "table-name": "%"
+            },
+            "filters": []
+        },
+        {
+            "rule-id": 733828293,
+            "rule-name": "mhdlhs8h-od97g1",
+            "rule-type": "transformation",
+            "rule-target": "table",
+            "rule-action": "rename",
+            "object-locator": {
+                "schema-name": "npd",
+                "table-name": "%"
+            },
+            "value": "npd-dms-test"
```

Comment:
We probably would not use this specific table name, but we continue to use something other than "npd" in the short term.

Terraform has support for interpolating placeholder values in JSON, which we can use to configure this value at the module level: https://developer.hashicorp.com/terraform/language/functions/templatefile